### DISCUSSION TITLE
How to edit a user bat file

### DISCUSSION THREAD

**Discussion by mindesu** [AbdBarho/stable-diffusion-webui-docker#588] [Q&A]
Hello, I'm very sorry to bother you.
I am using a translator, so please forgive me if I am using strange English.

I would like to use something like EasyNegative called unaestheticXL, but I found out that I have to add the following to webui-user.bat

add "git checkout dev" ,without the "" on an empty line below set COMMANDLINE_ARGS=

in webui-user.bat
How can I do that?

**Comment by DevilaN** [comment]
You can't. Those are instructions for windows webui starting script.

It seems that you need to update to latest development version of software, which is something I advise against of. If you still wish so, then pick latest commit and follow information provided here: https://github.com/AbdBarho/stable-diffusion-webui-docker/discussions/544
